### DISCUSSION TITLE
[Support]: HASS Intergration: Hide from Media Browser-Camera Entities?

### DISCUSSION THREAD

**Discussion by majorsl** [blakeblackshear/frigate#13487] [General Support]
### Describe the problem you are having

Not sure it is a bug, so I didn't file it in Issues for the HASS integration.  I don't want my Frigate items available in the Media Browser.  I found the settings aptly named "Enable the media browser" and unchecked it.  Frigate in there is empty, but I'm still seeing camera entities as supplied by Frigate.  Shouldn't those be gone as well?

### Version

0.14.1

### What browser(s) are you using?

Edge 128.0.2739.54

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

Debian

### Install method

Docker CLI

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

amcrest

**Comment by NickM-27** [comment]
No, the entities are unrelated to the media browser. You can disable the entities if you want

**Reply by majorsl** [reply]
Yes, but if I do that, then I cannot use them in the Frigate card.  I don't want certain users to be able to see them at all.  I control those permissions by views with the card in them or not.

What am I missing?  It seems like I'm doing something wrong.

I realize it doesn't help that HA devs seem completely uninterested in allowing that sidebar to have separate user settings.  It's silly.

**Reply by NickM-27** [reply]
Right, this isn't something that is controllable by the integration
